lib/api/v3/work_packages/work_package_representer.rb

```diff
@@ -391,6 +391,17 @@ def initialize(model, current_user:, embed_links: false)
                  end,
                  render_nil: true
 
+        property :duration,
+                 exec_context: :decorator,
+                 skip_render: ->(represented:, **) {
+                   represented.milestone? || !OpenProject::FeatureDecisions.work_packages_duration_field_active?
+                 },
+                 getter: ->(*) do
+                   datetime_formatter.format_duration_from_hours(represented.duration,
```

Comment:
There is a mismatch in units here. The duration column works in days since values smaller than a day are not possible. The getter currently treats the duration as if its value would be denoted in hours. The value thus needs to be multiplied by 24 to be correct. 

While not part of this PR, the column looks weird in the UI for a similar reason:

<img width="165" alt="image" src="https://user-images.githubusercontent.com/617519/172385328-9e0b19b3-a55d-4632-b8f4-63b0df0a0168.png">

Even if the value were to be displayed with their correct value, it is't readable to the end user like this. I forwarded that question to product.

Reply:
DONE: b9994104d1143a7676ae74b47c6c286c7edc8245

We can address the day format in a separate PR.